APE17.rst

```diff
@@ -500,10 +502,14 @@ and their proposed replacement:
 +-------------------------------------+-------------------------------------+
 
 We propose that a new package called extension-helpers be created starting from
-astropy-helpers but with only the minimal amount of functionality needed to
-handle the definition of compiled extensions, external libraries, and the
-auto-discovery of Cython extensions. This package would need to be declared as a
-build-time dependency in ``pyproject.toml``.
+astropy-helpers but with only functionality related to:
+
+* Collecting extension definitions from ``setup_package.py`` files
+* Auto-defining Cython extensions
+* Determining flags for OpenMP compilation
+
+This package would need to be declared as a build-time dependency in
+``pyproject.toml`` for packags that wish to use it.
```

Comment:
packages

Reply:
It would be great, if the `extension-helpers` could also figure out build/linker flags for C++ (if used), because this is sometimes a really pain, especially, if C++11 features are desired and one wants to build on multiple platforms.

Reply:
Indeed there's actually no reason that extension-helpers couldn't be extended to help with extensions beyond C (e.g. C++, rust, etc.) as long as we keep it general.